ophyd/controls/signal.py

```diff
@@ -423,20 +450,25 @@ def describe(self):
         dict
             Dictionary of name and formatted description string
         """
-        return {self.name: {'source': 'PV:{}'.format(self._read_pv.pvname),
-                            'dtype': 'number',
+        return {self.name: {'source': 'PV:%s' % self._read_pv.pvname,
+                            'dtype': self.dtype,
```

Comment:
If this is anything but 'number', the ophyd RunEngine will fall over. So far, I think, we have avoided inserting any change that would break the ophyd RunEngine.